packages/status/tests/php/test-status.php

```diff
@@ -293,6 +329,9 @@ public function get_is_staging_site_known_hosting_providers_data() {
 			'dreampress' => array(
 				'http://ebinnion.stage.site',
 			),
+			'newspack'   => array(
+				'http://test.newspackstaging.com',
+			),
```

Comment:
This is the actual test change.

Reply:
Looks good, thank you for making this happen.